src/components/LightningWallet.vue

```diff
@@ -842,7 +844,26 @@ export default {
       this.send.isSending = false;
     },
     async createInvoice() {
+      // Check that there is enough in the recieve
+      if (!this.maxRecieve || this.recieve.amount > this.maxRecieve) {
+        const toastOptions = {
+          title: "Low incoming balance",
+          autoHideDelay: 3000,
+          variant: "warning",
+          solid: true,
+          toaster: "b-toaster-top-center"
```

Comment:
The design system that we use has all the toasts on the bottom right, so I'd recommend the same placement for consistency:

```suggestion
          toaster: "b-toaster-bottom-right"
```